other/enforce-readwriteonce-pod/enforce-readwriteonce-pod.yml

```
@@ -0,0 +1,33 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: readwriteOnce-pod
```

```suggestion
  name: readwriteonce-pod
```

Can't have upper case letters in name.